velox/exec/StreamingWindow.cpp

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "velox/exec/StreamingWindow.h"
+#include "velox/exec/OperatorUtils.h"
+#include "velox/exec/Task.h"
+
+namespace facebook::velox::exec {
+
+void StreamingWindow::updatePartitions() {
```

Comment:
The logic of this method suggests that it starts a new partition. Perhaps, rename accordingly: startNewPartition or closePartition.

Reply:
Updated.